src/views/Drive.vue

```diff
@@ -1260,38 +1263,40 @@ module.exports = {
         },
 	    openInApp(args, app) {
                 if (app == null || app == "" || app == "Drive") {
+                    this.previouslyOpenedApp = {filename: '', app: ''};
                     this.closeApps();
-                    return
+                    return;
+                }
+                if (this.previouslyOpenedApp.filename == args.filename && this.previouslyOpenedApp.app == app) {

```

Comment:
I would expect opening a file with the same name in a different dir to be common, e.g. index.html